static/styles/_vars.scss

```diff
@@ -7,120 +7,4 @@ $lite-orange: #ff9900f2;
 $orange: #df8600f2;
 $wand-color: #ffd492;
 $button-height: 100px;
-
```

Comment:
Make sure you remove the unused code before merge

Reply:
Which part did you mean?

Reply:
I mean as a general rule, don't push code that shuoldnt be on development. There is a lot of commented code here that is being removed now.

Reply:
In some cases, I don't know how it happened ;). In others, something went on dev, and then the idea was modified, so it is the root of problems.